### PR TITLE
Migrate CI to GitHub Actions

### LINKED ISSUE #462: Migrate from Travis to GH Actions
Nice little walkthrough [here](https://www.youtube.com/watch?v=K4x-uqLl_m4&ab_channel=DeanAttali). Basically just need to call `usethis::use_github_actions()` and then do a bit of cleanup.

### PR DESCRIPTION
**Pull request by jwildfire**
# Overview 

This PR: 
- Sets up Github Action that runs package checks on PRs and Pushes to dev and master (surprisingly easy!)
- Clear all checks (not surprisingly a pain). 

Fixes #462 #488 

# Test notes

Make sure the app runs as expected and that all checks and tests pass locally.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -8,9 +8,8 @@ Authors@R: c(
 	person("Xiao","Ni", role = "aut"),
 	person("James","Buchanan", role="aut"),
 	person("Susan","Duke", role="aut"),
-	person("Interactive Safety Graphics Working Group", role = "aut"),
-	person("Rho Inc.", role = "cph"))
```
Comment (elimillera) [inline DESCRIPTION:12]: I believe we need to list the copyright holder from the LICENSE if its not going to be the maintainers. CRAN will reject a package if the copyright holder isn't noted in the description

**Review by elimillera — APPROVED** [review]
One comment in the DESCRIPTION but otherwise this looks great!

File: DESCRIPTION (hunk 1 above)
Comment (jwildfire) [reply DESCRIPTION:12]: Thanks for the heads up, @elimillera. I'm going to file a separate issue to deal with this and talk with the group about how to move forward.

**Comment by jwildfire** [comment]
I'm going to add `person("Interactive Safety Graphics Working Group", role = "cph")` unless someone has a better idea. 

@xni7 @JimBuchanan @mli1 @elimillera: thoughts?

**Pull request merged**